backend/sites/Dockerfile.migrations

```diff
@@ -21,5 +21,5 @@ COPY . .
 # Install postgresql-client
 RUN apk add --no-cache postgresql-client
 
-# Define the entrypoint
-ENTRYPOINT [ "sh", "initDB.sh" ]
\ No newline at end of file
+RUN chmod +x initDB.sh
+ENTRYPOINT ["/app/initDB.sh"]
```

Comment:
I'm curious if this is just a convention or if there's something about executing shell scripts in a docker container that I don't know about which can cause issues.